suite/tests/runcmp.cmake

```diff
@@ -111,5 +111,19 @@ if (lists_identical)
 endif (lists_identical)
 
 if (NOT lists_identical)
-  message(FATAL_ERROR "failed to match expected output")
+  find_program(DIFF_CMD diff)
+  if (DIFF_CMD)
+    # Detected diff command, print a full diff for better debugging.
+    list(JOIN output "" output)
+    list(JOIN filtered_expect "" filtered_expect)
+    set(tmp "${cmp}-out")
+    set(tmp2 "${cmp}-expect")
+    file(WRITE "${tmp}" "${output}")
+    file(WRITE "${tmp2}" "${filtered_expect}")
+    set(diffcmd "diff")
```

Comment:
I think this line should be removed and the execute_process should use `${DIFF_CMD}`, which is the one we found: right?

Reply:
Sorry, updated to use `DIFF_CMD`.